releaseNotes/KubernetesProvider.md

```diff
@@ -2,6 +2,15 @@
 
 [Image][image] | [Sample][sample]
 
+## 2.0.0 -
+
+### Enhancements
+* Added support for namespaced service account. It can establish federated identity credential between the workload identity and the namespaced service account subject, which is useful to control access to configuration between apps and namespaces.

```

Comment:
Added an enhancement that supports using custom service accounts for workload identity authentication. This is useful for permission management isolation between different namepsaces.